Tests/Test_BatteryBalancingSim.c

```diff
@@ -8,9 +8,12 @@
 
 cell_asic minions[NUM_MINIONS];
 
-int main() {
+void foo(){
```

Comment:
same thing as Michael's comment earlier, rename function

Reply:
same reply as to Michael's comment